worker/main.go

```diff
@@ -44,8 +43,13 @@ type Result struct {
 	Values    []float64
 }
 
+type task struct {
+	Graph      *graphs.Graph
+	Parameters map[string]string
+}
+
 var conf Config
-var graph graphs.Graph
+var tasks []task
```

Comment:
Voor thread safety kun je hier beter een channel geburuiken (dan is ook de sleep van 1 seconden in de `ProcessGraphsWhenAvailable()` niet nodig.